README.md

```diff
@@ -3,11 +3,11 @@ Budget Key data processing pipelines
 
 ## Quickstart
 ```bash
-$ pip install datapackage-pipelines
-$ export PYTHONPATH=`pwd`
-$ export DPP_PROCESSOR_PATH=`pwd`/processors/
-$ cd pipelines/
-$ dpp
+$ sudo apt-get install build-essential python3-dev libxml2-dev libxslt1-dev
+$ python --version
+Python 3.5.2+
+$ pip install -e .
+$ ./budgetkey-dpp
```

Comment:
@akariv - it should be available globally when you do `pip install -e .` from the setup.py console_scripts
I don't think it will work with the `./` prefix..